### PR TITLE
Log Instance Ids

### PR DESCRIPTION
**Pull request by sean-smith**
Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/utils.py [hunk 1]
```diff
@@ -85,6 +85,27 @@ def retrieve_cfn_resources(stack_name, region):
         raise
 
 
+def get_ec2_instances(stack_name, region):
```
Comment (demartinofra) [inline tests/integration-tests/utils.py]: minor: get_compute_nodes_instance_ids?

File: tests/integration-tests/utils.py [hunk 2]
```diff
@@ -85,6 +85,27 @@ def retrieve_cfn_resources(stack_name, region):
...
+            asg.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_name])
+            .get("AutoScalingGroups")[0]
+            .get("Instances")
+        )
+        instance_ids = []
+        for instance in instances:
+            instance_ids.append(instance.get("InstanceId"))
+        return instance_ids
+    except Exception as e:
+        logging.warning("Failed retrieving stack resources for stack {} with exception: {}".format(stack_name, e))
```
Comment (demartinofra) [inline tests/integration-tests/utils.py]: minor: logging.error

Comment (sean-smith) [reply tests/integration-tests/utils.py]: changed

File: tests/integration-tests/utils.py (hunk 1 above)
Comment (sean-smith) [reply tests/integration-tests/utils.py]: updated

**Pull request merged**